canvasapi/quiz.py

```diff
@@ -240,26 +244,165 @@ def get_all_quiz_submissions(self, **kwargs):
         Get a list of all submissions for this quiz.
 
         :calls: `GET /api/v1/courses/:course_id/quizzes/:quiz_id/submissions \
-        <https://canvas.instructure.com/doc/api/quiz_submissions.html#method.quizzes/quiz_submissions_api.index>`
+        <https://canvas.instructure.com/doc/api/quiz_submissions.html#method.quizzes/quiz_submissions_api.index>`_
 
         :rtype: list of :class:`canvasapi.quiz.QuizSubmission`
         """
         response = self._requester.request(
             'GET',
-            'courses/{}/quizzes/{}/submissions'.format(self.course_id, self.id),
+            'courses/{}/quizzes/{}/submissions'.format(
+                self.course_id,
+                self.id
+            ),
             _kwargs=combine_kwargs(**kwargs)
         )
         submission_list = response.json()['quiz_submissions']
 
         return [QuizSubmission(self._requester, submission) for submission in submission_list]
 
+    def get_quiz_submission(self, quiz_submission, **kwargs):
+        """
+        Get a single quiz submission.
+
+        :calls: `GET /api/v1/courses/:course_id/quizzes/:quiz_id/submissions/:id \
+        <https://canvas.instructure.com/doc/api/quiz_submissions.html#method.quizzes/quiz_submissions_api.show>`_
+
+        :param quiz_submission: The object or ID of the quiz submission to retrieve.
+        :type quiz_submission: int, string, :class:`canvasapi.quiz.QuizSubmission`
+
+        :rtype: :class:`canvasapi.quiz.QuizSubmission`
+        """
+        quiz_submission_id = obj_or_id(quiz_submission, "quiz_submission", (QuizSubmission,))
+
+        response = self._requester.request(
+            'GET',
+            'courses/{}/quizzes/{}/submissions/{}'.format(
+                self.course_id,
+                self.id,
+                quiz_submission_id
+            ),
+            _kwargs=combine_kwargs(**kwargs)
+        )
+
+        response_json = response.json()["quiz_submissions"][0]
+        response_json.update({'course_id': self.course_id})
+
+        return QuizSubmission(self._requester, response_json)
+
+    def create_submission(self, **kwargs):
+        """
+        Start taking a Quiz by creating a QuizSubmission can be used to answer
+        questions and submit answers.
+
+        :calls: `POST /api/v1/courses/:course_id/quizzes/:quizzes/:quiz_id/submissions \
```

Comment:
Remove the extra `:quizzes/`. URL should look like this:

```suggestion
        :calls: `POST /api/v1/courses/:course_id/quizzes/:quiz_id/submissions \
```